docs/api.rst

```diff
@@ -2048,7 +2048,7 @@ Request transactions event decoding
 
       {"async_query": true, "tx_hashes": ["0xe33041d0ae336cd4c588a313b7f8649db07b79c5107424352b9e52a6ea7a9742", "0xed6e64021f960bb40f11f1c00ec1d5ca910471e75a080e42b347ba5af7e73516"], "ignore_cache": false}
 
-   :reqjson list tx_hashes: The list of transaction hashes to request decoding for
+   :reqjson list tx_hashes: The list of transaction hashes to request decoding for. If no transaction hashes are passed, it decodes all transactions again ignoring the cache.
```

Comment:
The cache ignore or not should always be as per the given argument `ignore_cache`. Not this.

```suggestion
   :reqjson list tx_hashes: The list of transaction hashes to request decoding for. If no transaction hashes are passed, it implies all transactions
```